analyzer/codechecker_analyzer/analyzers/analyzer_types.py

```diff
@@ -136,6 +136,39 @@ def check_available_analyzers(analyzers, errored):
     sys.exit(1)
 
 
+def check_available_checkers(analyzer_config_map, ordered_checkers):
+    """
+    This function checks if any of the explicitly enabled/disabled checkers in
+    ordered_checkers is not supported by any analyzer. In this case program
+    execution stops.
+
+    TODO: This function is not the part of the final implementation of this
```

Comment:
We moved this PR to the 6.16.0 release so you will have time to implement it without ugly hacks.

Reply:
This will require an extensive refactoring, so I consider this as the final solution for now. I removed the TODO message.